.gitattributes

```diff
@@ -0,0 +1,7 @@
+# GitHub highlighting for Solidity files
+# See https://github.com/github/linguist/pull/3973#issuecomment-357507741
+*.sol linguist-language=Solidity
```

Comment:
Is this file needed? It's about solidity.

Reply:
removed the file